lib/textstat.rb

```diff
@@ -67,16 +67,14 @@ def self.avg_sentence_per_word(text)
   def self.flesch_reading_ease(text)
     sentence_length    = avg_sentence_length(text)
     syllables_per_word = avg_syllables_per_word(text)
-    flesch = (
-    206.835 - (1.015 * sentence_length).to_f - (84.6 * syllables_per_word).to_f
-    )
+    flesch = 206.835 - 1.015 * sentence_length - 84.6 * syllables_per_word
```

Comment:
operator precedence means that the brackets weren't needed, and `<float> * <integer>` will be a float so no need to cast it again